lib/ansible/modules/cloud/gandi/gandi_vps.py

```diff
@@ -377,7 +373,7 @@ def create_instances(module, driver, instance_names):
 
     lc_image = get_image(driver, image, lc_location)
     if not lc_image:
-        lc_image = get_volume(driver, image, lc_location)
+        lc_image = get_volume(driver, image)
```

Comment:
I think missing lc_location can break things internal to gandi.
The get_volume should accept an optional lc_location parameter, default it to None

Reply:
The call to the api in the libcloud module is never made with any optional parameters, should we also update it?

Reply:
yes we must propagate this optional parameter